explore-analyze/discover/discover-get-started.md

```diff
@@ -303,6 +305,35 @@ Note that in ES|QL mode, the **Documents** tab is named **Results**.
 
 Learn more about how to use ES|QL queries in [Using ES|QL](try-esql.md).
 
+### Run multiple explorations with tabs
+```{applies_to}
+stack: preview 9.2
+serverless: preview
+```
+**Discover** supports multiple tabs to help you explore different aspects of your data simultaneously. Each tab maintains its own independent state, including the query ({{esql}} or KQL), time range, filters, selected data source, columns and sort order you defined, and the active [context-aware experience](#context-aware-discover).
+
+This allows you to pivot quickly between different explorations without rebuilding everything. For example:
```

Comment:
```suggestion
This allows you to pivot quickly between different explorations without losing your place. For example:
```
Nit: might be more accurate since you technically don't have to rebuild things if you save and load Discover sessions, it's just a lot less convenient than tabs. I think this works either way though.